test/functional/api/vim_spec.lua

```diff
@@ -81,6 +81,26 @@ describe('api', function()
     end)
   end)
 
+  describe('nvim_execute_lua', function()
```

Comment:
why? how behaves it differently with call_atomic? Also this is to some extent intended as an _alternative_ to call_atomic in cases it is too primitive.